src/components/Header/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from 'wouter'
+import useUser from 'hooks/useUser'
+import { urlPath } from 'src/setup'
+
+export default function Header () {
+  const {isLogged, logout} = useUser()
+
+  const handleClick = e => {
+    e.preventDefault()
+    logout()
+  }
+
+  return (
+    <header>
+      {
+        isLogged
+        ? <Link href="#" onClick={handleClick}>Logout</Link>
```

Comment:
If you are not redirecting the user to anywhere, maybe you need a `button` instead of a link, it's semantically more accurate. If you still decide to go with a link, using the `!#` disables the route and helps you with SEO

```suggestion
        ? <Link href="!#" onClick={handleClick}>Logout</Link>
```